lib/resources/r4/related_person.yaml

```diff
@@ -131,25 +130,25 @@ fields:
               "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/period"
             },
             {
-              'valueCodeableConcept': {
-                'coding': [
+              "valueCodeableConcept": {
+                "coding": [
                   {
-                    'system': 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/40',
-                    'code': '155'
+                    "system": "http://terminology.hl7.org/CodeSystem/v3-RoleCode",
+                    "code": "SIS"
                   }
                 ]
               },
-              'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/relation'
+              "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/relation"
             }
           ]
         }
       ]
     }
   note: |
     <ul>
-      <li> The <code>relationship.coding</code> must contain exactly one proprietary system. </li>
-      <li> The <code>relationship.coding</code> cannot contain a standard system. </li>
       <li> When specifying a `period`, the fields must include a time component with a timezone. </li>
+      <li>Each `coding` can be either from the standard or proprietary system, but not both.</li>
+      <li>If the coding is from a proprietary system, only one code may be provided.</li>
```

Comment:
To be more accurate, I think we should update this line to state:
`If the coding contains a proprietary system, only one coding may be proved.`

Since there can only be one code per coding, but multiple codings per CodeableConcept.

cc. @alexbeckner1

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/662/commits/300867b2eb8e04f2456f4a36f5c45cc93172f23b#diff-261586474bd9faacdccf38c13eea61175a7bba5b7ce8ddb4dea273142eb9f6d6R151